metadata/src/test/java/org/apache/kafka/metalog/LocalLogManager.java

```diff
@@ -541,6 +541,8 @@ public void resign(int epoch) {
     @Override
     public Optional<SnapshotWriter<ApiMessageAndVersion>> createSnapshot(long committedOffset, int committedEpoch) {
         OffsetAndEpoch snapshotId = new OffsetAndEpoch(committedOffset + 1, committedEpoch);
+        return SnapshotWriter.createWithHeader(
+                () -> createNewSnapshot(snapshotId),
```

Comment:
There are two returns in a row. The second return is never reached.

Reply:
This was a miss. Let me fix it.